developer-docs-site/docs/nodes/validator-node/using-aws.md

```diff
@@ -14,8 +14,8 @@ Make sure you complete these pre-requisite steps before you proceed:
 1. Set up your AWS account. 
 2. Make sure the following are installed on your local computer:
 
-   * **Aptos CLI**: https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md
-   * **Terraform 1.1.7**: https://www.terraform.io/downloads.html
+   * **Aptos CLI 0.2.0**: https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md
```

Comment:
This doesn't match the CLI version

Reply:
Will release and update the link here, the release job is still running